src/main/java/com/mineboundteam/minebound/magic/DefensiveSpells/EarthDefensiveSpell.java

```diff
@@ -74,12 +76,29 @@ public void onUsingTick(ItemStack stack, Level level, Player player) {
                             breakProgress.put(player, data); 
 
                             if(data.getB() >= 1f && !level.isClientSide()){
+                                // Post the block break event
+                                BlockState state = level.getBlockState(blockpos);
+                                BlockEvent.BreakEvent event = new BlockEvent.BreakEvent(level, blockpos, state, player);
+                                MinecraftForge.EVENT_BUS.post(event);
+
+                                // Handle if the event is canceled
+                                if (event.isCanceled())
+                                    return;
+
                                 level.destroyBlock(blockpos, true);
                                 breakProgress.remove(player);
                                 reduceMana(config.MANA_COST_ON_CAST.get(), player);
                             }
                         }
                     } else if (!level.isClientSide()){
+                        // Post the block break event
+                        BlockState state = level.getBlockState(blockpos);
+                        BlockEvent.BreakEvent event = new BlockEvent.BreakEvent(level, blockpos, state, player);
+                        MinecraftForge.EVENT_BUS.post(event);
+
+                        // Handle if the event is canceled
+                        if (event.isCanceled())
+                            return;
                         level.destroyBlock(blockpos, true);
```

Comment:
Probably don't want the blocks to be dropped while in creative.

Reply:
Fixed